core/modules/utils/linked-list.js

```diff
@@ -95,6 +95,15 @@ LinkedList.prototype.toArray = function() {
 	return output;
 };
 
+LinkedList.prototype.tiddlerIterator = function(wiki) {
```

Comment:
I wonder if this method should be `makeTiddlerIterator` to match the corresponding wiki store method?

Reply:
Fair enough. It does return a method which behaves identically (and must behave identically) to the $tw.wiki.makeTiddlerIterator. I'd considered making the wikimethod handle it instead, but I think it better fits the grain if LinkedList has its own, since it also has its own "pushTop" method.

Change is made, and PR is good to go.